app/models/like.rb

```diff
@@ -1,4 +1,4 @@
 class Like < ApplicationRecord
-  belongs_to :author
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
   belongs_to :post
 end
```

Comment:
- [ ] [REQUIRED] Let's also make sure that the Like model has a method that updates the likes counter for a post as stated in the project requirement.